icicle/include/icicle/api/koalabear.h

```diff
@@ -0,0 +1,109 @@
+// WARNING: This file is auto-generated by a script.
+// Any changes made to this file may be overwritten.
+// Please modify the code generation script instead.
+// Path to the code generation script: scripts/gen_c_api.py
+
+#pragma once
```

Comment:
@ChickenLover @nonam3e do we even need those files? I suggest removing them. it's probably never used and no reason to use them.

Reply:
agree

Reply:
ok, deleted them

Reply:
sorry, didn't noticed that current examples depend on these files. It's better to remove them in another pr

Reply:
@yshekel we need to discuss about it removing them, and in another PR ofc